mongo/src/main/java/org/axonframework/mongo/eventsourcing/eventstore/StorageStrategy.java

```diff
@@ -120,4 +120,15 @@ void appendSnapshot(MongoCollection<Document> snapshotCollection, DomainEventMes
      * @param snapshotsCollection The collection containing the document representing snapshots
      */
     void ensureIndexes(MongoCollection<Document> eventsCollection, MongoCollection<Document> snapshotsCollection);
+
+    /**
+     * Return the last known sequence number for an Aggregate with given {@code aggregateIdentifier}, whose Events are
+     * stored in the given {@code eventsCollection}.
+     *
+     * @param eventsCollection    The Collection in which to search for Events
```

Comment:
Nit: Shouldn't there be dots at the end of the `@param` and `@return` descriptions?